features/config.feature

```diff
@@ -92,6 +92,43 @@ Feature: Config
       The requested config file does not exist
       """
 
+  Scenario: PHP configuration file
+    Given a file named "behat.php" with:
+      """
+      <?php
+
+      return new Behat\Config\Config();
+
+      """
+    And a file named "features/bootstrap/FeatureContext.php" with:
+      """
+      <?php
+
+      use Behat\Behat\Context\Context;
+
+      class FeatureContext implements Context
+      {
+      }
+      """
+    And a file named "features/config.feature" with:
+      """
+      Feature:
+        Scenario:
+          When this scenario executes
+      """
+    When I run "behat -f progress --no-colors --append-snippets --config=behat.php"
```

Comment:
Do we need to specify a config file here? Shouldn't Behat be able to pick it up automatically?

Reply:
Exactly, I was based on previous scenarios.

Reply:
done